src/coreclr/vm/threadsuspend.cpp

```diff
@@ -3069,7 +3079,7 @@ BOOL Thread::RedirectCurrentThreadAtHandledJITCase(PFN_REDIRECTTARGET pTgt, CONT
     if (srcFeatures != 0)
     {
 #if defined(TARGET_X86) || defined(TARGET_AMD64)
-        const DWORD64 xStateFeatureMask = XSTATE_MASK_AVX | XSTATE_MASK_AVX512;
+        const DWORD64 xStateFeatureMask = XSTATE_MASK_AVX | XSTATE_MASK_AVX512 | XSTATE_MASK_APX;
```

Comment:
This is Windows only code. I think that SetXStateFeaturesMask will fail if we pass in an unknown mask.

Reply:
It explicitly allows unknown masks, so that you don't need to do version/build specific checks (<https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-setxstatefeaturesmask>):
> The system silently ignores any feature specified in the FeatureMask which is not enabled on the processor.

Reply:
I see.